### DISCUSSION TITLE
Is it possible to display the date on certain pages without overriding the template?

### DISCUSSION THREAD

**Discussion by emslade** [imfing/hextra#176] [Q&A] [answered]
Hi,

As the title, I'm looking to display the date on some pages without overriding the template:

For example on https://docs.gamebench.net/docs/release-notes/private-server/v2-22-1/.

As I see it, my options are:

* Include the date in the content of the file (duplicating the date in the front matter)
* Modify the single page template to conditionally include the date on the page

Is there another way I can achieve this?

Thank you!

**Comment by imfing** [comment]
Hi @emslade , thanks for using Hextra, the site looks great!

By "display the date", do you mean showing the date similar to what we do in the blog post:

<img width="731" alt="image" src="https://github.com/imfing/hextra/assets/5097752/2ae1bb33-24e1-45fc-8bc4-032f96b42d3e">

**Reply by emslade** [reply]
Thank you for developing the theme!

Yes that's correct, I didn't want to use the blog functionality though.

Thanks!

**Reply by imfing** [reply, marked as answer]
some other ways I can think of:

For example, define a new layout `single-with-date.html` and explicitly set the front matter:

```yaml
title: example
date: 2020-02-20
layout: single-with-date
```

---

Alternatively, you may also define a custom shortcode `layouts/shortcodes/date.html`, and include that on the top of your content which reads the date from the front matter

```markdown
---
title: example
date: 2020-02-20
---

{{< date >}}

Rest of the content...
```

**Reply by emslade** [reply]
Great! Thank you
